### ISSUE TITLE
openssl 1.1.1b compile failed on macOS Mojave: can't find inttypes.h?

### ISSUE THREAD

**Issue by yt7589** [openssl/openssl#8996]
OS: macOS Mojave
Xcode: 10.1
openssl: 1.1.1b

when I build openssl:
`./config`
without any problem.
`make`
It reported errors as bellow:

```
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" \
	    "-oMakefile" crypto/include/internal/bn_conf.h.in > crypto/include/internal/bn_conf.h
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" \
	    "-oMakefile" crypto/include/internal/dso_conf.h.in > crypto/include/internal/dso_conf.h
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" \
	    "-oMakefile" doc/man7/openssl_user_macros.pod.in > doc/man7/openssl_user_macros.pod
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" \
	    "-oMakefile" include/openssl/opensslconf.h.in > include/openssl/opensslconf.h
/usr/bin/perl "-I." -Mconfigdata "util/dofile.pl" \
	    "-oMakefile" test/provider_internal_test.conf.in > test/provider_internal_test.conf
/Library/Developer/CommandLineTools/usr/bin/make depend && /Library/Developer/CommandLineTools/usr/bin/make _all
clang  -I. -Iinclude -Iapps/include  -fPIC -arch x86_64 -O3 -Wall -DL_ENDIAN -DOPENSSL_PIC -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_BN_ASM_GF2m -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DKECCAK1600_ASM -DRC4_ASM -DMD5_ASM -DAES_ASM -DVPAES_ASM -DBSAES_ASM -DGHASH_ASM -DECP_NISTZ256_ASM -DX25519_ASM -DPOLY1305_ASM -DOPENSSLDIR="\"/usr/local/ssl\"" -DENGINESDIR="\"/usr/local/lib/engines-3\"" -DMODULESDIR="\"/usr/local/lib/ossl-modules\"" -D_REENTRANT -DNDEBUG  -MMD -MF apps/libapps-lib-app_rand.d.tmp -MT apps/libapps-lib-app_rand.o -c -o apps/libapps-lib-app_rand.o apps/app_rand.c
In file included from apps/app_rand.c:10:
In file included from apps/include/apps.h:13:
In file included from ./e_os.h:16:
In file included from include/openssl/e_os2.h:243:
/Users/arxanfintech/Desktop/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/10.0.0/include/inttypes.h:30:15: fatal error: 'inttypes.h' file not found
#include_next <inttypes.h>
              ^~~~~~~~~~~~
1 error generated.
make[1]: *** [apps/libapps-lib-app_rand.o] Error 1
make: *** [all] Error 2
```

**Comment by slontis** [comment]
@levitte this looks related to the conversation we are having in #8986

**Comment by slontis** [comment]
@yt7589  can you try looking at https://github.com/ray-project/ray/issues/1861.

**Comment by agnosticdev** [comment]
I've run into a similar, but not the exact `inttypes.h`, issue when building with Xcode's 10.1 toolchain.  I just grabbed a 10.2 toolchain and ran a test build with -b `OpenSSL_1_1_1-stable` and all went fine.  
Xcode 10.2
Build version 10E125

Until this issue is resolved do you have access to a 10.2 toolchain to try a build?

**Comment by mattcaswell** [comment]
This question seems to have been answered. Closing.
